### ISSUE TITLE
Crash when I press and release 2 buttons at the same time on touchscreen

### ISSUE THREAD

**Issue by AngryCarrot789** [AvaloniaUI/Avalonia#15636]
### Describe the bug

When I press 2 buttons at once and release, there's a likely chance it crashes my application

### To Reproduce

Press 2 buttons with 2 fingers, then release both quickly

### Expected behavior

It should handle the "pointer releases" successfully

### Avalonia version

11.0.10

### OS

Linux

### Additional context

Here's the stack trace of the crash:
```
Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at Avalonia.Input.Gestures.PointerReleased(RoutedEventArgs ev)
   at Avalonia.Reactive.LightweightObservableBase`1.PublishNext(T value)
   at Avalonia.Interactivity.Interactive.RaiseEvent(RoutedEventArgs e)
   at Avalonia.Input.TouchDevice.ProcessRawEvent(RawInputEventArgs ev)
   at Avalonia.Input.InputManager.ProcessInput(RawInputEventArgs e)
   at Avalonia.ManualRawEventGrouperDispatchQueue.DispatchNext()
   at Avalonia.Controls.Platform.ManagedDispatcherImpl.RunLoop(CancellationToken token)
   at Avalonia.Threading.DispatcherFrame.Run(IControlledDispatcherImpl impl)
   at Avalonia.Threading.Dispatcher.PushFrame(DispatcherFrame frame)
   at Avalonia.Threading.Dispatcher.MainLoop(CancellationToken cancellationToken)
   at Avalonia.Controls.DesktopApplicationExtensions.Run(Application app, CancellationToken token)
   at LinuxFramebufferPlatformExtensions.StartLinuxDirect(AppBuilder builder, String[] args, IOutputBackend outputBackend, IInputBackend inputBackend)
   at BlueberryBenchSupply.Program.Main(String[] args)
```

**Comment by timunie** [comment]
can we test 11.1 beta?

**Comment by maxkatz6** [comment]
Yes, a very similar problem was fixed in 11.1 beta2

**Comment by AngryCarrot789** [comment]
Seems to be fixed in 11.1+ like you said. Sweet!
